src/Compilers/CSharp/Test/Symbol/Symbols/DefaultInterfaceImplementationTests.cs

```diff
@@ -50528,7 +50528,7 @@ public class C2 : I1
                 Diagnostic(ErrorCode.ERR_BadMemberFlag, "P1").WithArguments("abstract").WithLocation(9, 21),
                 // (9,26): error CS8051: Auto-implemented properties must have get accessors.
                 //     abstract int I1.P1 { set; }
-                Diagnostic(ErrorCode.ERR_AutoPropertyMustHaveGetAccessor, "set").WithArguments("C2.I1.P1.set").WithLocation(9, 26)
+                Diagnostic(ErrorCode.ERR_AutoPropertyMustHaveGetAccessor, "set").WithArguments("C2.I1.P1.set").WithLocation(9, 26)s
```

Comment:
This doesn't look like an intentional change.

Reply:
Yeah My <kbd>CTRL</kbd>+<kbd>s</kbd> fails 😆